crates/transport-http/Cargo.toml

```diff
@@ -26,5 +26,5 @@ hyper = { workspace = true, features = ["full"], optional = true }
 
 [features]
 default = ["reqwest"]
-reqwest = ["dep:reqwest", "dep:alloy-json-rpc", "dep:serde_json", "dep:tower"]
+reqwest = ["dep:reqwest", "dep:alloy-json-rpc", "dep:serde_json", "dep:tower", "reqwest/default-tls"]
```

Comment:
I think I would prefer this as follows:

```
default = ["reqwest", "default-tls"]
reqwest = ["dep:reqwest", "dep:alloy-json-rpc", "dep:serde_json", "dep:tower"]

default-tls = ["reqwest?/default-tls"]
native-tls = ["reqwest?/native-tls"]
rustls-tls = ["reqwest?/rustls-tls"]
```

Reply:
that way the TLS backend is still fully configurable, without changing the default behavior from your version

Reply:
makes sense, amended with your suggestion